crates/biome_js_analyze/tests/specs/correctness/noUnusedImports/invalid-unused-react.jsx.snap

```diff
@@ -19,7 +20,7 @@ import React, { useEffect } from "x"
 ```
 invalid-unused-react.jsx:1:8 lint/correctness/noUnusedImports  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 
-  ! This import is unused.
+  ! These imports are unused.
```

Comment:
The message *was* correct. Maybe we could use a more broad message, for example, "Unused import(s) detected."

Reply:
Fixed :)